packages/opentelemetry-sdk-node/README.md

```diff
@@ -54,24 +54,14 @@ const sdk = new opentelemetry.NodeSDK({
 
   // See the Configuration section below for additional  configuration options
 });
-
-// You can optionally detect resources asynchronously from the environment.
-// Detected resources are merged with the resources provided in the SDK configuration.
-sdk
-  .start()
-  .then(() => {
-    // Resources have been detected and SDK is started
-  })
+// You can add you own resources which will be merged with autodetected resources from the environment.
+sdk.start();
 ```
 
 ## Configuration
 
 Below is a full list of configuration options which may be passed into the `NodeSDK` constructor;
 
-### autoDetectResources
```

Comment:
Are we sure we want to remove this option completely? While I think most will want to autodetect resources, I could see wanting to opt-out in some environments.